src/Main.java

```diff
@@ -0,0 +1,22 @@
+public class Main {
+    public static void main(String[] args) {
+        Sorting sort = new Sorting();
+
+        sort.add(new Player("Robert", 100));
+        sort.add(new Player("Johny", 234));
+        sort.add(new Player("MutherFucker666", 512));
```

Comment:
This is a public repository of your github account and possible employers can look at this code. Consider better naming next time.